package.json

```diff
@@ -4,16 +4,17 @@
   "description": "A travel booking server",
   "main": "index.js",
   "scripts": {
-    "pre-test": "cross-env NODE_ENV=test npm run migrate",
-    "test": "cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha src/test/index.test --require @babel/register --recursive --exit || true",
+    "pre-test": "cross-env NODE_ENV=test npm run undo-migrate && npm run migrate",
```

Comment:
why did you add `undo-migrate`

Reply:
It was because of the changes made to the existing migrations. We had to undo all migrations and run migrate again.